### PR TITLE
add workshop participants endpoint

### PR DESCRIPTION
**Pull request by PetrDlouhy**
Participants can be found on URL like `/api/workshops/1/participants`

### CODE REVIEW

File: improtreskApi/rest_router.py [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+
+router = NestedDefaultRouter()
 router.register(r'lectorRoles', lector_roles.LectorRoleViewSet)
 router.register(r'lectors', lectors.LectorViewSet)
 router.register(r'news', news.NewsViewSet)
 router.register(r'texts', texts.TextViewSet)
 router.register(r'workshopDifficulties', workshop_difficulties.WorkshopDifficultyViewSet)
 router.register(r'years', years.YearViewSet)
 router.register(
+    r'workshops',
```
Comment (just-paja) [inline improtreskApi/rest_router.py]: `/workshops` were removed in favor of /years/x/workshops. Please move this under `/years/x/workshops/y/participants/`

File: api/rest/participants.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            'email',
+            'phone',
+            'rules_accepted',
+            'newsletter',
+            'paid',
+            'assigned_workshop',
+        )
+
+
+class ParticipantViewSet(NestedViewSetMixin, viewsets.ReadOnlyModelViewSet):
```
Comment (just-paja) [inline api/rest/participants.py]: Needs authentication on admin level

Comment (PetrDlouhy) [inline api/rest/participants.py]: Now that is ensured by `permission_classes` param.

File: improtreskApi/rest_router.py (hunk 1 above)
Comment (PetrDlouhy) [inline improtreskApi/rest_router.py]: This was fixed.

**Comment by PetrDlouhy** [comment]
@just-paja Fixed all your issues.

**Comment by just-paja** [comment]
@PetrDlouhy, thanks, but there is one more.

This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.

**Pull request merged**